addons/mail/models/res_partner.py

```diff
@@ -26,6 +26,7 @@ class ResPartner(models.Model):
     # we need this to be readable inline as tracking messages use inline HTML nodes
     contact_address_inline = fields.Char(compute='_compute_contact_address_inline', string='Inlined Complete Address', tracking=True)
     starred_message_ids = fields.Many2many('mail.message', 'mail_message_res_partner_starred_rel')
+    im_status = fields.Char("IM Status", compute="_compute_im_status")
```

Comment:
I'd like to remove the field from partner, it makes no sense. It's only a feature on users.
But maybe this can be done as a follow up PR. Once I clean Store of user/partner/persona this might be much easier to do.